derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -686,11 +719,32 @@ impl<'a> FieldWithDefaults<'a> {
     }
 
     pub fn field_vis(&self) -> Visibility {
-        self.field
-            .field
-            .as_expressed_vis()
-            .or_else(|| self.parent.field.as_expressed_vis())
-            .unwrap_or(Visibility::Inherited)
+        if !self.field_enabled() {
```

Comment:
Why does this now preempt the `#[builder(field(...))]` option?

Reply:
I have added a comment about this.  This preserves the behaviour from before my MR.  Previously this private visibility was dealt with in ad-hoc code in the builder struct code generator.  Now, the field *type* comes from `BuilderFieldType`, and the code generator always uses the visibility which comes from `field_vis`, so `field_vis` must return the desired answer.

Reply:
First: Great comment.

Second: I agree with removing the ad-hoc hiding in codegen, so let's keep the logic for visibility all here.

Third: With #247, I feel like explicit field-level visibility should overrule the inferred visibility from being a phantom. How would you feel about this checking `self.field.field.as_expressed_vis()`, then checking if the field is enabled, then checking the struct-level? Or should it check those first with the normal inheritance chain and finally fall back to inferring `Inherited` vis if nothing is pushing to make it public?

Reply:
> I feel like explicit field-level visibility should overrule the inferred visibility

Good point.  I agree completely. about explicitly-specified field-level visibility,  But, I think, only field-level, not struct-level visibility.